### PR TITLE
Amisiura bulk merge rewrite

### PR DESCRIPTION
**Pull request by AntonMisiura**

### CODE REVIEW

File: Extension/Sencilla.Repository.EntityFramework.Extension/Bootstrap.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿global using System.Text;
```
Comment (vitalykovalgit) [inline Extension/Sencilla.Repository.EntityFramework.Extension/Bootstrap.cs]: Remove namespace from folder name

File: Repository/EntityFramework/Repository/CreateRepository.cs [hunk 2]
```diff
@@ -1,101 +1,86 @@
-﻿
-namespace Sencilla.Repository.EntityFramework
+﻿using Sencilla.Repository.EntityFramework.Extension;
```
Comment (vitalykovalgit) [inline Repository/EntityFramework/Repository/CreateRepository.cs]: Remove using

File: Sencilla.EntityFramework.Extension.Tests/Sencilla.Repository.EntityFramework.Extension.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (vitalykovalgit) [inline Sencilla.EntityFramework.Extension.Tests/Sencilla.Repository.EntityFramework.Extension.Tests.csproj]: remove namespace from folder path and move to test folder

File: Extension/Sencilla.Repository.EntityFramework.Extension/Sanitizer/SqlSanitizer.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+﻿namespace Sencilla.Repository.EntityFramework.Extension;
+
+public static class SqlSanitizer
+{
+    private static readonly string _pattern = @"(\b(ALTER|CREATE|DELETE|DROP|EXEC(UTE)?|INSERT( +INTO)?|MERGE( +INTO)?|SELECT|UPDATE( +TOP)?|UNION( +ALL)?|WHERE|ORDER( +BY)?|GROUP( +BY)?|FROM|HAVING|CASE( +WHEN)?|AND|OR|LIKE|LIMIT|OFFSET)\b)|(--.*)|(/\*(.|[\r\n])*?\*/)";
```
Comment (vitalykovalgit) [inline Extension/Sencilla.Repository.EntityFramework.Extension/Sanitizer/SqlSanitizer.cs]: try to find another solution

File: Extension/Sencilla.Repository.EntityFramework.Extension/Provider/QueryProvider.cs [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        return tr.Condition;
+    }
+
+    public string ToSqlParameterValue(PropertyInfo p, object ov)
+    {
+        if (ov == null)
+            return $"NULL";
+
+        if (p.PropertyType == typeof(string))
+            return $"N'{SqlSanitizer.Sanitize(ov.ToString())}'";
```
Comment (vitalykovalgit) [inline Extension/Sencilla.Repository.EntityFramework.Extension/Provider/QueryProvider.cs]: Check on this case (when string contains ' char)
param' delete * from sec.users

File: Core/Repository/ICreateRepository.cs
Comment (AntonMisiura) [inline Core/Repository/ICreateRepository.cs:1]: плвіфплдіфдпоівфолпілвп

File: Core/Repository/ICreateRepository.cs [hunk 6]
```diff
@@ -40,27 +39,14 @@ public interface ICreateRepository<TEntity, TKey> : IReadRepository<TEntity, TKe
...
-    Task<TEntity?> Upsert(TEntity entity,
-        System.Linq.Expressions.Expression<Func<TEntity, object>> columnPrimaryKeyExpression, CancellationToken token = default);
-
-    /// <summary>
-    /// Upsert many entities async
-    /// </summary>
-    /// <param name="entity"></param>
-    /// <returns></returns>
-    Task<IEnumerable<TEntity>> Upsert(System.Linq.Expressions.Expression<Func<TEntity, object>> columnPrimaryKeyExpression, params TEntity[] entities);
-
```
Comment (vitalykovalgit) [inline Core/Repository/ICreateRepository.cs:58]: Let's add these methods as well

**Pull request merged**